pages/DSNP/UserData.md

```diff
@@ -190,12 +191,12 @@ The following example illustrates the output of a Get User Data Operation invoca
       {
         "data": base64_string,
         "etag": string,
-        "keyId": integer
+        "keyIndex": integer
```

Comment:
Would this be a change to the https://github.com/LibertyDSNP/graph-sdk/ that needs to happen?

Reply:
I don't think this "example" JSON serialization is used directly in graph-sdk, but @aramikm or Joe might know better. I'll get some more eyes on it. If there's prior art out there using `keyId` I'd be fine with keeping it that way for simplicity's sake, but documenting that `keyId` is the index.

Reply:
Per Joe this won't impact graph-sdk, so resolving this thread.

Reply:
After thinking about this, I think this should just keep `keyId`. It is an identifier even if it applies an index algo.

Reply:
Agreed